web/styles/portico/portico_signin.css

```diff
@@ -1581,3 +1581,46 @@ button#register_auth_button_gitlab {
 .self-hosting-login-help-text {
     margin-top: 10px;
 }
+
```

Comment:
I wonder if it's worth adding some explanatory comments to this CSS--especially if altcha's upstream structures/styles were to change and we needed to readjust accordingly...clearly there's been a lot of thoughtful work here on these adjustments, and it would be bad for that to be lost/forgotten.